ImageProcessingRequest_api

```diff
@@ -0,0 +1,86 @@
+v1.ImageProcessingRequest
+
+This is an object that describes request to the ChRIS system to be able to process an image into a desired format
+
+Fields:
+    - imageprocessingrequestid: an autogenerated id that corresponds to the submitted job
+    - patientid: The id of the patient
+    - jobtype: The type of image processing job you want to run
+    - jobstatus: The status of the job (e.g. 'Pending', 'Running', 'Failed', 'Success')
```

Comment:
do we need `jobArguments []string` to allow people to pass additional configuration information to a job?